docs/reference/keyboard-shortcuts.md

```diff
@@ -26,3 +35,17 @@ The following keyboard shortcuts can currently be used:
  - **F2**: Rename node
  - **Shift + ArrowLeft**: Select all nodes left of the current one
  - **Shift + ArrowRight**: Select all nodes right of the current one
+
+## Editor canvas zoom (>= 0.126.0)
+- **+** or **=**: Zoom in
+- **-** or **_**: Zoom out
+- **0**: Reset zoom level
+- **1**: Zoom to fit workflow
+- **Ctrl + Mouse wheel**: Zoom in/out
+
+## Editor canvas zoom (< 0.126.0)
```

Comment:
Remove this section. Documentation will always reflect the most current release.